### PR TITLE
Base input and accordion

### PR DESCRIPTION
**Pull request by brandonreid**
Updated styles utilizing tokens for the base input and accordion components

### CODE REVIEW

**Review by znicholasbrown — COMMENTED** [review]
Is `hue` the right word for this token? Reading through these tokens it looks like we're saying:
color-focus-color. Perhaps something like focus-highlight?

File: src/styles/index.css [hunk 1]
```diff
@@ -161,6 +165,20 @@
...
+    --p-color-input-text: var(--p-color-text-default);
+    --p-color-input-placeholder: var(--p-color-text-subdued);
+
+    --p-color-input-bg-invalid: var(--p-color-input-bg);
+    --p-color-input-border-invalid: var(--p-color-sentiment-negative);
+    --p-color-input-text-invalid-icon: var(--p-color-sentiment-negative);
+
+    --p-color-input-activated-bg: var(--p-color-active);
+    --p-color-input-activated-border: var(--p-color-active);
+    --p-color-input-activated-text: var(--p-color-text-default);
```
Comment (brandonreid) [inline src/styles/index.css:180]: The activated input tokens are for checked checkbox/radios. Accidentally added those, can remove if we wanna.

Comment (znicholasbrown) [reply src/styles/index.css:180]: Can we swap that for `active` then? I believe that's the CSS semantic term instead of `activated`

Comment (brandonreid) [reply src/styles/index.css:180]: I think you're thinking of `checked`, which I can swap for that here.. I can't imagine another input type that would have a sort of activated state, so..

Comment (brandonreid) [reply src/styles/index.css]: Updated!

Comment (znicholasbrown) [reply src/styles/index.css]: Ah yeah you're right, sorry. Yea I'm mostly trying to make sure we can map tokens onto their semantic meanings for consistency. We'll be grateful for that later I think

**Review by znicholasbrown — APPROVED** [review]

File: src/styles/index.css (hunk 1 above)
Comment (brandonreid) [reply src/styles/index.css]: Totally! More than welcomed!

**Pull request merged**